ai_genomics/pipeline/patent_data/evaluation_sample.py

```diff
@@ -0,0 +1,111 @@
+"""ai_genomics/pipeline/patent_data/evaluation_sample.py
+
+Tool for exporting a sample of
+"""
+
+import click
+import pandas as pd
+from pathlib import PosixPath
+
+from ai_genomics import PROJECT_DIR
+
+
+def ai_genomics_counts(
```

Comment:
question - are you using cookiecutter's precommit? When I do, it reformats my scripts to _not_ have new lines for the parameters passed in a function.